scripts/fast-reboot

```diff
@@ -15,9 +15,10 @@ then
 fi
 
 # Kernel and initrd image
-KERNEL_IMAGE="/vmlinuz"
-INITRD="/initrd.img"
-BOOT_OPTIONS=$(cat /proc/cmdline)
+NEXT_SONIC_IMAGE=$(sonic_installer list | grep "Next: " | cut -d ' ' -f 2)
+BOOT_OPTIONS=$(cat /host/grub/grub.cfg | sed "/$NEXT_SONIC_IMAGE'/,/}/"'!'"g" | grep linux)
```

Comment:
On my test machine previous version of BOOT_OPTIONS was
`BOOT_IMAGE=/image-HEAD.230-cce2a36/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda4 rw console=tty0 console=ttyS1,9600n8 quiet loop=image-HEAD.230-cce2a36/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor varlog_size=4096 fast-reboot`

Your version of the same variable is
`linux /image-HEAD.230-cce2a36/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda4 rw console=tty0 console=ttyS1,9600n8 quiet loop=image-HEAD.230-cce2a36/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor varlog_size=4096`

So difference is s/BOOT_IMAGE=/linux     /

I've checked your script with my fast-reboot test. The test passes.

So honestly, I don't know this "BOOT_IMAGE" is critical or not.
So I'll approve.
But it would be nice of you to convert BOOT_OPTIONS to "BOOT_IMAGE=" format.

Reply:
Please convert your BOOT_OPTIONS to "BOOT_IMAGE" format. You can do this by appending 
`| sed -e 's/\s*linux\s*/BOOT_IMAGE=/'` into your BOOT_OPTIONS. (after grep linux).
Also you need to adjust your KERNEL_IMAGE option

Reply:
it worked... but you're right need to fix this